pulumi/src/docker.ts

```diff
@@ -59,7 +59,9 @@ export const buildAndPushImage = async (args: BuildAndPushImageArgs): Promise<vo
 
   dockerBuildArgs.push(args.context)
 
-  execSync(`${envs} docker build --platform linux/amd64 ${dockerBuildArgs.join(' ')}`, { stdio: 'inherit' })
+  execSync(`${envs} docker build --platform linux/amd64 -o type=docker ${dockerBuildArgs.join(' ')}`, {
```

Comment:
just curious, are you manually changing these or do you have some formatting config set in your IDE doing this? have been noticing a lot of these the last couple prs

Reply:
Ugh, docker is pain. Turns out that if you install just the docker engine, not docker desktop (as I just did on my linux box) it doesn't generate a docker image automatically, so I've had to add it explicitly.

Reply:
oh, i didn't even notice the type addition, i was just asking about the formatting of the objects on multiple lines vs single line.

are you using docker buildx? I am on linux without docker desktop and haven't seen this issue. it looks like this bundles docker archive images and I'm not sure we want this. feels like more of a local environment issue that needs to be resolved.

Reply:
kk will revert for now